ShaiRandom.UnitTests/BasicTests.cs

```diff
@@ -101,9 +101,9 @@ public void NextDecimalDistributionTest()
             {
                 buckets[(int)(r.NextDecimal() * 256)]++;
             }
-            IOrderedEnumerable<int> ob = buckets.OrderBy(b => b);
-            int smallest = ob.First();
-            int biggest = ob.Last();
+            Array.Sort(buckets);
```

Comment:
Ah, this is in a test; I was wondering where I used LINQ in the library code itself... I guess here!